cfr.json

```diff
@@ -397,7 +397,7 @@
           "label": {
             "string_id": "spotlight-total-cookie-protection-header"
           },
-          "size": "20px"
+          "size": "23px"
```

Comment:
is the size change intended?

Reply:
Yeah, I tested on a bunch of locales and zoom now that https://bugzilla.mozilla.org/show_bug.cgi?id=1767606 was uplifted to release 100.0.1. Victoria had concerns that the existing title and subtitle size/spacing were too similar.